docs/sdk/src/guides/handling_parachain_forks.rs

```diff
@@ -0,0 +1,87 @@
+//! # Parachain forks
+//!
+//! In this guide, we will examine how AURA-based parachains handle forks. AURA (Authority Round) is
+//! a consensus mechanism where block authors rotate at fixed time intervals. Each author gets a
+//! predetermined time slice during which they are allowed to author a block. On its own, this
+//! mechanism is fork-free.
+//!
+//! However, since the relay chain provides security and serves as the source of truth for
+//! parachains, the parachain is dependent on it. This relationship can introduce complexities that
+//! lead to forking scenarios.
+//!
+//! ## Background
+//! Each parachain block has a relay parent, which is a relay chain block that provides context to
+//! our parachain block. The constraints the relay chain imposes on our parachain can cause forks
+//! under certain conditions. With asynchronous-backing enabled chains, the node side was building
+//! blocks on all relay chain forks. This meant that no matter which fork of the relay chain
+//! ultimately progressed, the parachain would have a block ready for most of them. The situation
+//! changes when parachains want to produce blocks at a faster cadence. In a scenario where a
+//! parachain might author on 3 cores with elastic scaling, it is not possible to author on all
+//! relay chain forks. This limitation necessitates a strategic approach to building blocks.
+//!
+//! ## Impact of Forks
+//! When a relay chain fork occurs and the parachain builds on a fork that will not be extended in
+//! the future, the blocks built on that fork are lost and need to be rebuilt. This increases
+//! latency and reduces throughput, affecting the overall performance of the parachain.
```

Comment:
This is the case when collator choosed the wrong fork right? And this case is strongly coupled with beforementioned inablitily to build on all relayc forks?
Am I getting this right?